src/tweak.h

```diff
@@ -40,37 +40,37 @@ class CTweakBase
 inline void fill(const UniValue &v, double &output)
 {
     if (v.isStr())
-        output = boost::lexical_cast<double>(v.get_str());
+        output = std::stod(v.get_str());
```

Comment:
I'd rather have this change and all the changes below on a separate PR.

Reply:
taken out of this PR and already merged in 1.6.0.1